src/components/RegistrationForm/RegistrationForm.jsx

```diff
@@ -32,28 +32,34 @@ const RegistrationForm = () => {
     const confirm = form.elements.confirm.value;
     const name = form.elements.name.value;
 
-    if (password.length >= 6 && password.length <= 12) {
-      console.log('pswrd ok');
+    if (email.includes('@') && email.length >= 3) {
+      Notiflix.Notify.success('Email OK');
     } else {
-      console.log('pswrd bad');
+      Notiflix.Notify.failure(
+        'Email has to include "@" and be at least 3 characters long!'
+      );
     }
-    if (email.includes('@') && email.length >= 3) {
-      console.log('email ok');
+    if (password.length >= 6 && password.length <= 12) {
+      Notiflix.Notify.success('Password OK');
```

Comment:
brakuje jeszcze warunku że musi zawierać znak ale nie mam pojęcia jak to napisać. Przepuszczam